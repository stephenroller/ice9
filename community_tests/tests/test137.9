# first example from spec clarification					succeed


var a : int				# uses level 0, built-in
type int = bool
var b : int				# uses level 1, global

a := 1;
b := true;

write a;
write ?b;