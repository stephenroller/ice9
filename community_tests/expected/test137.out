Loading file: generated/test137.tm
1 
1 
Number of instructions executed = 28
